tests/test_query.py

```diff
@@ -377,3 +377,69 @@ async def test_trivial_unbatching(redis):
         },
         output["message"]
     )
+    
+@pytest.mark.asyncio
+@with_translator_overlay(
+    settings.kpregistry_url,
+    settings.normalizer_url,
+    kp_data={
+        "kp0":
+        """
+            MONDO:0008114(( category biolink:Disease ))
+            MONDO:0008114-- predicate biolink:related_to -->MESH:C035133
+            MESH:C035133(( category biolink:Gene ))
+        """,
+        "kp1":
+        """
+            MESH:C035133(( category biolink:Gene ))
+            MESH:C035133-- predicate biolink:related_to -->HP:0007430
+            HP:0007430(( category biolink:Protein ))
+            MESH:C035133-- predicate biolink:related_to -->HP:0007431
```

Comment:
What is the purpose of this?

Reply:
Good catch. I actually forgot that I put that in there. That was an incomplete part of the test that I must have forgotten about. I'll get that fixed now.